api4/team.go

```diff
@@ -369,6 +369,8 @@ func getTeamMember(c *Context, w http.ResponseWriter, r *http.Request) {
 }
 
 func getTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
+	sort := r.URL.Query().Get("sort")
```

Comment:
Can we push this down below `c.RequireTeamId` to help emphasize the "order" of parameters here?

Reply:
+1 also `excludeDeletedUsers` after `c.RequireTeamId()`

Reply:
Thanks! Done.